dev-libs/pigpio/pigpio-66.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit distutils-r1 systemd
+
+DESCRIPTION="pigpio is a library for the Raspberry which allows control of the GPIOs"
+HOMEPAGE="http://abyz.me.uk/rpi/pigpio/index.html"
+SRC_URI="https://github.com/joan2937/pigpio/archive/V${PV}.tar.gz"
+
+LICENSE="Unlicense"
```

Comment:
this is not possible, packages without license are not permitted

Reply:
The Unlicense is a public domain equivalent license with a focus on an anti-copyright message.
https://en.wikipedia.org/wiki/Unlicense

Reply:
Ah, totally forgot about this one